node_modules/oae-core/editprofile/js/editprofile.js

```diff
@@ -111,9 +111,25 @@ define(['jquery', 'oae.core'], function ($, oae) {
             }
 
             // Render the form elements
+            // detect if is possible to modify the name and email
+            var disableModification = false;
+            var authenticationStrategy = oae.data.me.authenticationStrategy;
+           if ((authenticationStrategy === 'cas')| (authenticationStrategy === 'shibboleth')){
```

Comment:
I think this should be `if ((authenticationStrategy === 'cas') || (authenticationStrategy === 'shibboleth'))`